metrics/report/makereport.sh

```diff
@@ -23,6 +23,12 @@ GUESTINPUTDIR="/inputdir/"
 GUESTOUTPUTDIR="/outputdir/"
 
 setup() {
+	echo "Checking subdirectories"
+	check_subdir="$(cd ${HOSTINPUTDIR}; ls -dx */ | wc -l)"
+	if [ $check_subdir -eq 0 ]; then
+		die "Subdirectory not found"
```

Comment:
Maybe we need to be more explicit here? 
What kind of subdirectory does this script needs? How can it be created?